config-default.yml

```
@@ -122,6 +122,7 @@ guild:
     channels:
         announcements:                              354619224620138496
         user_event_announcements:   &USER_EVENT_A   592000283102674944
+        python_news:                &PYNEWS_CHANNEL 701667765102051398
```

A channel has been made, you can change this ID:
```suggestion
        python_news:                &PYNEWS_CHANNEL 704372456592506880
```